stubs/protobuf/google/protobuf/internal/well_known_types.pyi

```diff
@@ -63,29 +67,33 @@ class _FieldMaskTree:
     def AddLeafNodes(self, prefix: tAny, node: tAny) -> None: ...
     def MergeMessage(self, source: tAny, destination: tAny, replace_message: tAny, replace_repeated: tAny) -> None: ...
 
+_StructValue: TypeAlias = struct_pb2.Struct | struct_pb2.ListValue | str | float | bool | None
+_StructValueArg: TypeAlias = _StructValue | Mapping[str, _StructValueArg] | Sequence[_StructValueArg]
+
 class Struct:
-    def __getitem__(self, key: tAny): ...
-    def __contains__(self, item: tAny): ...
-    def __setitem__(self, key: tAny, value: tAny) -> None: ...
-    def __delitem__(self, key: tAny) -> None: ...
+    def __getitem__(self, key: str) -> _StructValue: ...
+    def __contains__(self, item: _StructValueArg) -> bool: ...
```

Comment:
Assuming `self.fields` is some kind of mapping, this should work with any object:

```suggestion
    def __contains__(self, item: object) -> bool: ...
```

Using `object` as the annotation here is in line with what we do with other mapping classes in typeshed: https://github.com/python/typeshed/blob/1c940e89141e0c09eeebbce2679760ec03173beb/stdlib/typing.pyi#L586